.gitignore

```diff
@@ -58,3 +58,5 @@ target/
 
 # pyLCI
 /config.json 
+/pylci.log*
+do_not_load
```

Comment:
Ok. I was thinking it would be a nice feature to have this as a part of config rather than a file dropped in the director. The work was out of scope for this feature though.